### PR TITLE
dynamic labels

### PR DESCRIPTION
**Pull request by gaodeng**
Usage scenarios :  change i18n locale without restart the app.

### CODE REVIEW

**Comment by sindresorhus** [comment]
You need to add a test and docs updates in readme and index.d.ts

File: readme.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Show the `Look Up [selection]` menu item when right-clicking text on macOS.
 
 #### labels
 
-Type: `Object`<br>
+Type: `Object | Function`<br>
```
Comment (caesar) [inline readme.md]: I think some more detailed information in the readme on how this is intended to be used would be helpful.

Comment (gaodeng) [reply readme.md]: updated

File: fixture.js [hunk 2]
```diff
@@ -4,14 +4,16 @@ const {app, BrowserWindow} = require('electron');
...
+	labels: (event, props) => {
+		return {
+			cut: 'Configured Cut',
+			copy: `Configured Copy ⎡ ${props.selectionText} ⎦ `,
+			paste: 'Configured Paste',
+			save: 'Configured Save Image',
+			saveImageAs: 'Configured Save Image As…',
+			copyLink: 'Configured Copy Link',
+			inspect: 'Configured Inspect'
+		};
```
Comment (sindresorhus) [inline fixture.js]: Don't overload the existing fixture. Create a new one.

File: readme.md [hunk 3]
```diff
@@ -117,6 +117,17 @@ Format:
 	}
 }
 ```
+or
+
+```js
```
Comment (sindresorhus) [inline readme.md]: This needs to include the `labels` key too, so it's easy to compare.

File: readme.md [hunk 4]
```diff
@@ -117,6 +117,17 @@ Format:
...
+(event, props) => {
```
Comment (sindresorhus) [inline readme.md]: It's not clear to the user what `props` is unless they use TS.

**Comment by sindresorhus** [comment]
The PR needs a better title. See other PRs for how to write a good title.

**Pull request closed without merging**